more_itertools/recipes.py

```diff
@@ -1075,3 +1077,89 @@ def totient(n):
     for prime in set(factor(n)):
         n -= n // prime
     return n
+
+
+# Miller–Rabin primality test: Testing against small sets of bases
+# https://w.wiki/CGwr
```

Comment:
For the runs of small primes (most of the entries in the table), a more authoritative source is https://oeis.org/A014233